scripts/start-deployLeaf

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+# shellcheck source=start-utils
+. "${SCRIPTS:-$(dirname "$0")}/start-utils"
+set -o pipefail
+set -e
+isDebugging && set -x
+
+resolveVersion
+: "${LEAF_BUILD:=LATEST}"
+
+# Docs at https://api.leafmc.one/docs/swagger-ui/index.html
+leafApiUrl="https://api.leafmc.one/v2/projects/leaf"
+leafDownloadsPage="https://www.leafmc.one/download"
+
+if ! get --exists "${leafApiUrl}/versions/${VERSION}/builds"; then
+  logError "Leaf builds do not exist for ${VERSION}"
+  logError "       check ${leafDownloadsPage} for available versions"
+  logError "       and set VERSION accordingly"
+  exit 1
+fi
+
+if [[ "${LEAF_BUILD^^}" == "LATEST" ]]; then
+  # Get the latest build number from the API, which will be the last object in the builds array
+  if ! buildNumber=$(
+    get --json-path '$.builds[-1].build' "${leafApiUrl}/versions/${VERSION}/builds"
```

Comment:
I pushed a set of changes to fix up the URL path and response structure. It was just slightly different than the others.

Reply:
If you want to see what I pushed

https://github.com/itzg/docker-minecraft-server/pull/3470/commits/284dec4e2b942fec494358a13f7bcc13193ae4f6